### DISCUSSION TITLE
Is it possible to colorize a menu?

### DISCUSSION THREAD

**Discussion by faerot** [elfmz/far2l#2449] [Q&A]
I have created a plugin which scans the file and presents it's summary in form of classes / functions declaration like this:

![image](https://github.com/user-attachments/assets/964363eb-1422-4554-b037-e640cb8944bc)

What I want is to highlight function names with different color so it will be more prominent.
I have tried mutiple & signs like &n&a&m&e, but it highlights only first letter.
Any way to hack this?

**Comment by ctapmex** [comment]
The color of the menu bar cannot be set. This is not present in Far3 (windows) either.

> I have created a plugin which scans the file and presents it's summary in form of classes / functions declaration

try to press the keys in the editor:  F11-F-5
Maybe you didn't know about it.

**Reply by faerot** [reply]
> The color of the menu bar cannot be set. This is not present in Far3 (windows) either.
> 
> > I have created a plugin which scans the file and presents it's summary in form of classes / functions declaration
> 
> try to press the keys in the editor: F11-F-5 Maybe you didn't know about it.

The actual plugin is not important, it is intermediate step for other goals, the question was about highlighting.
